build.sh

```diff
@@ -1,75 +1,145 @@
 #!/bin/bash
 
-# Script to build valkey-json module, build it and generate .so files, run unit and integration tests.
-
-# # Exit the script if any command fails
 set -e
 
-SCRIPT_DIR=$(pwd)
-echo "Script Directory: $SCRIPT_DIR"
+function print_usage() {
+cat<<EOF
+Usage: build.sh [--release] [--unit] [--integration] [--clean]
+
+    --help | -h               Print this help message and exit.
+    --release                 Builds the release configuration.
+    --unit                    Builds the unit tests configuration.
+    --integration             Builds the integration tests configuration.
+    --clean                   Cleans the build artifacts.
+
+Example usage:
+
+    # Build the release configuration,
+    ./build.sh --release
 
-if [ "$1" = "clean" ]; then
-  echo "Cleaning build artifacts"
-  rm -rf build/
-  echo "Clean completed."
-  exit 0
+    # Cleans the build artifacts,
+    ./build.sh --clean
+
+EOF
+}
+
+SCRIPT_DIR=$(pwd)
+BUILD_DIR="$SCRIPT_DIR/build"
+RUN_UNIT=1
+RUN_INTEGRATION=1
+RELEASE_BUILD=0
+CLEAN_BUILD=0
+
+## Parse command line argument
+while [[ $# -gt 0 ]]; 
+do
+    arg="$1"
+    case $arg in
+        --release)
+            RELEASE_BUILD=1
```

Comment:
```suggestion
            BUILD_RELEASE=1
```

Probably needs to be changed in multiple places.